### ISSUE TITLE
How to start a Parse Server with Express using npm package parse-server?

### ISSUE THREAD

**Issue by vegidio** [ParsePlatform/parse-server#1402]
I installed and managed to run properly Parse Server **with Express** using the example server that you have on Github (https://github.com/ParsePlatform/parse-server-example).

This is what I did:

1. I cloned the parse-server-example repo.
2. Edited the file `index.js` based on my needs.
3. Then I ran `npm install` and `npm start`.

Now I want to achieve the same thing using the package **parse-server** available on npm, but based on the instructions that I found in the package page (https://www.npmjs.com/package/parse-server) I couldn't figure out how to run it **with Express**. I managed to run it only without Express.

In other words, I don't want to clone the parse-server-example every time I need to use Parse. I want to start my server using only the `parse-server` command line tool and the `index.js` file.

Thanks

**Comment by flovilmart** [comment]
This is fairly easy;

1. Create an empty director
2. In that dir run `npm init`
3. Run `npm install --save parse-server`
4. In package.json add ```{ ..., "scripts": { "start": "parse-server config.json"}}```
5. Create a config.json with your option (appId, masterKey...)
6. Run `npm start`

**Comment by vegidio** [comment]
Thanks, but how can I make this approach read the `index.js` file that contains the Express code?
